tests/Microsoft.Identity.Test.Integration.netfx/Infrastructure/SeleniumExtensions.cs

```diff
@@ -24,8 +25,9 @@ public static class SeleniumExtensions
 
         public static IWebDriver CreateDefaultWebDriver()
         {
-            ChromeOptions options = new ChromeOptions();
-            ChromeDriver driver;
+            EdgeOptions options = new EdgeOptions();
```

Comment:
Because of this, don't we need to update the classic pipeline yaml to install the Edge driver too?

Reply:
No need, they keep the edge driver in sync on the hosted agents just like the chrome driver
https://github.com/actions/runner-images/blob/main/images/windows/Windows2022-Readme.md#browsers-and-drivers